packages/flutter/lib/src/services/debug.dart

```diff
@@ -25,6 +25,9 @@ KeyDataTransitMode? debugKeyEventSimulatorTransitModeOverride;
 /// The statistics include the total bytes transmitted and the average number of
 /// bytes per invocation in the last quantum. "Up" means in the direction of
 /// Flutter to the host platform, "down" is the host platform to flutter.
+///
+/// Contrary to other debugging toggles in this package, this will function even
+/// when [kReleaseMode] is true.
 bool debugProfilePlatformChannels = false;
```

Comment:
You'll have to remove the `debug` prefix from the variable.  Things with that prefix should be guaranteed to only run in debug builds.

Reply:
My interpretation of the "debug" prefix was that it's more of a debugging toggle - I couldn't find any other variables in any of the `debug.dart` files in the framework that don't have the prefix. On the other hand, perhaps this is the first toggle that we want to be usable in release mode.

Reply:
It's my understanding that anything with the debug prefix should be tree-shaken in release builds.  Maybe we'll have to move it out of the `debug.dart` file in that case.

Reply:
Sure, I moved it to platform_channel.dart